src/ol_concourse/pipelines/open_edx/mfe/pipeline.py

```diff
@@ -40,6 +40,7 @@ class OpenEdxVars(BaseModel):
     accessibility_url: Optional[str] = None
     account_settings_url: Optional[str] = None
     contact_url: Optional[str] = None
+    certificate: Optional[str] = None
```

Comment:
Should this be a boolean that we coalesce to a string in the build, or at least a `Literal['true', 'false']`? Setting it as a `str` makes it hard to know what the valid values are.

Reply:
I would also suggest that the variable be renamed to something like `enable_certificate_page` so that it's clearer when only seeing the variable what its purpose is.